src/python/grpcio_tests/tests/interop/server_ssl_cert_reload_test/_test.py

```diff
@@ -0,0 +1,411 @@
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import atexit
+import os
+import subprocess
```

Comment:
Why does this test have to use multiple processes? Why can't the new feature be tested in a single process?

Reply:
i originally had these separate programs for "demo"/manual testing before looking into gRPC tests. the feature doesn't have to be tested in separate processes.

to test in a single process, can you please point me to an example that can control two clients? do i use/extend something like `ServerClientMixin` in `cygrpc_test.py`?

Reply:
looks like there's the `_secure_intraop_test.py` that i can build off

Reply:
done